internal/vfs/kind_string.go

```diff
@@ -0,0 +1,16 @@
+// Code generated by "stringer -type=Kind -linecomment=true"; DO NOT EDIT.
```

Comment:
I know its a generated file but we might want to have the license header added.

Reply:
Done. I updated the `go:generate` command to run `beatsfmt` on the generated file. Not perfect, but it's the a good way to ensure I don't drop the license header in another PR, as I tend to run `go:generate` from time to time.

beatsfmt puts the license header at the top. The side effect of this is that github does not detect the `Code generated by` comment anymore -> diff will be shown.